packages/remix-dev/compiler/plugins/urlImportsPlugin.ts

```diff
@@ -0,0 +1,15 @@
+import type { Plugin } from "esbuild";
+
+/**
+ * Mark all URL imports as external so that the URL import is preserved in the build output.
+ */
+export const urlImportsPlugin = (): Plugin => {
+  return {
+    name: "url-imports",
+    setup(build) {
+      build.onResolve({ filter: /^https?:\/\// }, ({ path }) => {
```

Comment:
Shouldn't we make this more specific, so we _really_ only support https://deno.land & https://esm.sh ? 🤔

Reply:
No, Deno could import from arbitrary URLs, not just those ones.

Reply:
Your commit description made it sound like we only wanted to support these official ones, that's why I was a bit confused I guess.